src/components/content.js

```diff
@@ -342,6 +343,7 @@ class Content extends React.Component {
     const data = {}
     data.type = 'fragment'
     data.fragment = state.fragment
+    data.inVoidNode = state.fragment.nodes.some(node => node.isVoid)
```

Comment:
Hey @oyeanuj why do we need to add this here, instead of just performing this `some` check inside the `onCutOrCopy` handler? I think it would be cleaner to do it that way if possible.